### ISSUE TITLE
TabContainer: Sometimes fails to manage focus in Chrome

### ISSUE THREAD

**Issue by rigdern** [winjs/winjs#554]
### Repro Steps

1. [Download the repro](http://1drv.ms/1pfVYkO)
2. Open repro/index.html in Chrome
3. Hit the tab key

**Expected:** Focus leaves "1" and skips past elements "2", "3", and "4"

**Actual:** Focus goes to element "2" (it is orange which indicates that it has focus)

The green elements ("1", "2", "3", "4") have a TabContainer around them so you should not be able to tab between them. The `childFocus` is "1".

Repros in Chrome. Works okay in Firefox and IE.

If you change one of these things, the bug will stop reproing:
- Put a focusable element after the element which hosts the TabContainer.
- Instead of using the TabContainer to manage the focus of a bunch of buttons, use it to manage the focus of a bunch of focusable divs.

**Comment by xirzec** [comment]
This doesn't repro for me on the latest Chrome
